Views/SubscribedEvents/Search/global.html.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+?>
+<?php if (!empty($showMore)): ?>
+    <a href="<?php echo $view['router']->generate('mautic_contactclient_index', ['search' => $searchString]); ?>"
+       data-toggle="ajax">
+        <span><?php echo $view['translator']->trans('mautic.core.search.more', ['%count%' => $remaining]); ?></span>
+    </a>
+<?php else: ?>
+    <a href="<?php echo $view['router']->generate(
+        'mautic_contactclient_action',
+        ['objectAction' => 'view', 'objectId' => $client->getId()]
+    ); ?>" data-toggle="ajax">
+        <span><?php echo $client->getName(); ?></span>
+        <?php
```

Comment:
Extra  <?php ?>   ?

Reply:
I removed this. Not sure if It undoes the review or not.